src/App.js

```diff
@@ -320,7 +338,9 @@ const AppWithReduxStore = () => <AppWithRedux store={store} />;
 
 const AppWithSentry = Sentry.wrap(AppWithReduxStore);
 
-const AppWithCodePush = codePush(AppWithSentry);
+const codePushOptions = { checkFrequency: codePush.CheckFrequency.MANUAL };
```

Comment:
We might wanna have a diff config for prod, right?

Reply:
mmm, not really! We check this in `initSentry` (which will be renamed) so we didn't change the functionality of checking on each app start. I verified this behavior and works as was working before!